src/app/containers/StakePage/index.tsx

```diff
@@ -553,7 +553,8 @@ interface FeeProps {
 function FeeBlock({ contractToken, usdTotal }: FeeProps) {
   const account = useAccount();
   const { t } = useTranslation();
-  const token = (contractToken.asset + '_token') as any;
+  const token = (contractToken.asset +
+    (contractToken.asset === Asset.SOV ? '_token' : '_itoken')) as any;
```

Comment:
Dapp has itoken contracts sufixed with _lending:

```suggestion
    (contractToken.asset === Asset.SOV ? '_token' : '_lending')) as any;
```